esm_runscripts/batch_system.py

```diff
@@ -196,28 +196,31 @@ def calculate_requirements(config, cluster = None):
         if cluster in reserved_jobtypes:
             for model in config["general"]["valid_model_names"]:
                 omp_num_threads = int(config[model].get("omp_num_threads", 1))
-                if (
-                    config[model].get("nprocar", "remove_from_namelist") != "remove_from_namelist" 
-                    and 
-                    config[model].get("nprocbr", "remove_from_namelist") != "remove_from_namelist" 
-                    ):
-                        config[model]["tasks"] = config[model]["nprocar"] * config[model]["nprocbr"]
-                
-                elif "nproca" in config[model] and "nprocb" in config[model]:
-                    config[model]["tasks"] = config[model]["nproca"] * config[model]["nprocb"]
+
+                if "nproca" in config[model] and "nprocb" in config[model]:
+                    config[model]["tasks"] = \
+                        config[model]["nproca"] * config[model]["nprocb"]
                     #end_proc = start_proc + int(config[model]["nproca"])*int(config[model]["nprocb"]) - 1
+                    # KH 30.04.20: nprocrad is replaced by more flexible
+                    # partitioning using nprocar and nprocbr
+                    no_nprocr = ["remove_from_namelist", 0]
```

Comment:
OK, after reading this twice, I understand what this is doing. Maybe put some comments? Or maybe a better name: `remove_nprocr_options` or something

Reply:
Thanks for the review :) , I just added the commits and I'll merge now to `prep_release`.